docs/user-guide/discovery-and-relay.md

```diff
@@ -1,16 +1,15 @@
 # Deploying the Nexodus Discovery and Relay Nodes
```

Comment:
Pull `Discovery` from the header if you dont mind.

Reply:
Thanks for catching that. Fixed.